PERSONAL_DATA_DISCLOSURE.md

```diff
@@ -0,0 +1,93 @@
+## Overview
+
+The purpose of this form is to disclose the types of personal data[^1] (PD) stored by each module.
+This information enables those hosting FOLIO to better manage and comply with various privacy laws
+and restrictions, e.g. GDPR.
+
+It's important to note that PD is not limited to that which can be used to identify a person on it's
+own (e.g. Social security number), but also data used in conjunction with other data to identify a
+person (e.g. date of birth + city + gender, or dynamic IP address + date + time), and any
+information about such a person (e.g. has loaned 5 items, 2 of which are overdue).
+
+For the purposes of this form, "store" includes the following:
+
+- Persisting to storage - Either internal (e.g. Postgres) or external (e.g. S3, etc.) to FOLIO
+- Caching - In-memory, etc.
+- Logging
+- Sending to an external piece of infrastructure such as a queue (e.g. Kafka), database (e.g.
+  Elasticsearch, Library Data Platform), distributed table, etc.
+
+## Personal Data processed by this Module
+
+- [x] This module does not store any PD.
+- [ ] This module does not process any PD.
```

Comment:
The absence of a mark here suggests that personal data is processed by the module yet no information is marked below.

Reply:
Resolved (hopefully) as part of the addition of metadata